L1Trigger/TrackFindingTracklet/src/DTCLink.cc

```diff
@@ -0,0 +1,29 @@
+#include "L1Trigger/TrackFindingTracklet/interface/DTCLink.h"
+
+using namespace std;
+using namespace Trklet;
+
+DTCLink::DTCLink(double phimin, double phimax) {
+  if (phimin > M_PI) {
```

Comment:
I don't think this simplifies the code because if we use this separately on phimin and phimax we would then have to add code to make sure that phimin and phimax are consistent, i.e. ordered.

Reply:
This was left as unresolved, by answered by @aryd above.